ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -134,6 +135,53 @@ if tflite_support_is_available
   )
 endif
 
+if tflite2_support_is_available
+  filter_sub_tflite2_sources = ['tensor_filter_tensorflow_lite.cc']
+
+  nnstreamer_filter_tflite2_sources = []
+  foreach s : filter_sub_tflite2_sources
+    nnstreamer_filter_tflite2_sources += join_paths(meson.current_source_dir(), s)
+  endforeach
+
+  nnstreamer_filter_tflite2_deps = tflite_support_deps + [thread_dep, libdl_dep, glib_dep, gst_dep, nnstreamer_dep]
+
+  tflite2_compile_args = []
+
+  if cxx.has_type('kTfLiteInt8', prefix : '#include <tensorflow2/contrib/lite/model.h>')
+    tflite2_compile_args += '-DTFLITE_INT8=1'
+  endif
+  if cxx.has_type('kTfLiteInt16', prefix : '#include <tensorflow2/contrib/lite/model.h>')
+    tflite2_compile_args += '-DTFLITE_INT16=1'
+  endif
+  if cxx.has_type('kTfLiteFloat16', prefix : '#include <tensorflow2/contrib/lite/model.h>')
+    tflite2_compile_args += '-DTFLITE_FLOAT16=1'
+  endif
+  if cxx.has_type('kTfLiteComplex64', prefix : '#include <tensorflow2/contrib/lite/model.h>')
+    tflite2_compile_args += '-DTFLITE_COMPLEX64=1'
```

Comment:
IMO, tflite2 supports all those data types. Therefore, in the case of tfilte2, we do not need theses cxx.has_type statements.